packages/nextjs/src/client/components/presentation/SignIn/SignIn.tsx

```diff
@@ -103,21 +92,16 @@ const SignIn: FC<SignInProps> = ({
   const handleOnSubmit = async (
     payload: EmbeddedSignInFlowHandleRequestPayload,
     request: EmbeddedFlowExecuteRequestConfig,
-  ): Promise<EmbeddedSignInFlowHandleResponse> => await signIn(payload, request);
-
-  const handleError = (error: Error): void => {
-    onError?.(error);
+  ): Promise<EmbeddedSignInFlowHandleResponse> => {
+    return await signIn(payload, request);
   };
 
   return (
     <BaseSignIn
+      // isLoading={isLoading || !isInitialized}
```

Comment:
Is this change expected?

Reply:
This will be properly fixed in a upcoming PR.